docs/UserGuide.md

```diff
@@ -188,7 +191,41 @@ What you should see:
 
 ![hint](./images/hint.png)
 
-### 3.9. Displaying stats: `stats`
+
+### 3.9. Adding a note: `note add`
+
+Adds a note to a specified topic.
+
+Note:
+- There can be many notes under each topic
+- The UI will guide you through the note-adding process
+
+Example of usage:
+```
+note add
+```
+
+What you should see:
+
+### 3.10. Deleting a note: `note delete`
+
+Deletes a note from a specific topic.
+
+Example of usage:
+```
+note delete
+```
+
+### 3.11. Listing out notes: `note list`
+
+Lists out all notes belonging to a specific topic.
+
+Example of usage:
+```
+note list
+```
```

Comment:
Good update to the user guide, not sure if these sections should be nested under a notes feature